spyder/plugins/workingdirectory/tests/test_workingdirectory.py

```diff
@@ -7,14 +7,38 @@
 
 # Test library imports
 import pytest
+import os
+import os.path as osp
 
 # Local imports
 from spyder.plugins.workingdirectory.plugin import WorkingDirectory
+from spyder.config.base import get_home_dir
+from spyder.config.main import CONF
+
+
+NEW_DIR = 'new_workingdir'
 
 
 @pytest.fixture
-def setup_workingdirectory(qtbot):
-    """Setupp working directory plugin."""
+def setup_workingdirectory(qtbot, request):
+    """Setup working directory plugin."""
+    try:
+        use_startup_wdir = request.node.get_closest_marker('use_startup_wdir')
```

Comment:
With `get_closest_marker` you don't need to enclose this in a try/except (see PR #9818).

Please also add `use_startup_wdir` tp the [pytest.ini](https://github.com/spyder-ide/spyder/blob/master/pytest.ini) markers, with a description.